src/error-code.ts

```diff
@@ -1314,6 +1314,40 @@ export namespace ErrorCodes {
     type: ErrorType.ERROR,
     title: "Step Function Retry Invalid Input.",
   };
+
+  /**
+   * Step Functions Arithmetic Only Supports Integer
+   *
+   * Step Functions only supports integer addition via the `States.MathAdd`.
+   *
+   * ```ts
+   * new StepFunction(stack, "sfn", async (input: { a: number }) => {
+   *    return 1.5 + input.a;
+   * });
+   * ```
+   *
+   * If the above machine is given input: `{ a: 0.5 }`, the result will be `1`.
+   *
+   * Effectively resulting in: `Math.floor(1.5) + Math.floor(0.5)` => `1`
```

Comment:
I think this can be described more simply. 

> Step Functions can only add integers. When adding floating point numbers, the service will floor each of the numbers before adding them. 

```
1.5 + 1.5 = 2
```